src/locales/ro/datepicker.js

```diff
@@ -0,0 +1,3 @@
+const format = 'DD/MM/YYYY';
```

Comment:
For "ro" Firefox suggests DD.MM.YYYY as format. It would be more consistent to use the same, but if Firefox is wrong, fine to also accept it as it is here. Thoughts?

Reply:
The `DD/MM/YYYY` is the most common used form in Romanian. Wikipedia also [suggests](https://ro.wikipedia.org/wiki/Wikipedia:Date_%C8%99i_numere) using this format for Romanian content.